.circleci/config.yml

```diff
@@ -23,8 +23,8 @@ commands:
     steps:
       - restore_cache:
           keys:
-            - <<parameters.python>>-creme_crm-{{ checksum "setup.cfg" }}
-            - <<parameters.python>>-creme_crm
+            - <<parameters.python>>-creme_crm-v2.2-{{ checksum "setup.cfg" }}
```

Comment:
I'm not confortable with this "static" way to name the cache. I assume that will be once in a while, so fine, but can you add a TODO about find a better way to generate the checksum (maybe extracting the version from git last tag or whatever ?)

Reply:
In fact I shouldn't have used the creme version here. What we need is a static version or name that we can use to manually "invalidate" the cache, like I had to with those image updades. But using the creme version as part of the cache key is also a good idea and may be implemented later.
Another option is to stop falling back on a defaut cache (the second restore cache, which use the later one), and so always rely on the setup.cfg hash